packages/network-of-terms-catalog/catalog/datasets/goudatijdmachine-straten.jsonld

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://www.goudatijdmachine.nl/id/straten",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Goudse straten"
+    }
+  ],
+  "creator": [
+    {
+      "@id": "https://www.goudatijdmachine.nl",
+      "@type": "http://schema.org/Organization",
+      "name": "Gouda Tijdmachine",
+      "alternateName": "GTM"
+    }
+  ],
+  "url": [
+    "https://www.goudatijdmachine.nl/data/referentie/straten"
```

Comment:
This URL should be the URI prefix for terms in this dataset. Is this value correct? It seems the dataset uses URIs like [https://n2t.net/ark:/60537/b9MTpG](https://www.goudatijdmachine.nl/sparql/resource?uri=https%3A%2F%2Fn2t.net%2Fark%3A%2F60537%2Fb9MTpG) instead.